src/core/cuda_wrapper.hpp

```diff
@@ -22,6 +22,7 @@
 #if defined(__CUDACC__)
 
 #include <cuda.h>
+#include "clang_thrust_workaround.cuh"
```

Comment:
The file `clang_thrust_workaround.cuh`. Also, I just found out the issue was fixed in thrust 1.9.5 (I'm writing a report for them)
```suggestion
#if THRUST_VERSION < 100905
#include "clang_thrust_workaround.cuh"
#endif
```

Reply:
Here is my investigation: https://github.com/thrust/thrust/issues/1032#issuecomment-611617047; if CUDA 10.1 is shipped with thrust 1.9.5, we should install it on ICP machines and drop support for the combination Clang+CUDA10.0 is ESPResSo.

Reply:
@RudolfWeeber sorry, I forgot that editing a message doesn't send an email. In my previous message, I meant to say that you probably forgot to commit file `clang_thrust_workaround.cuh`.